fred2/briefingeditordlg.cpp

```diff
@@ -521,11 +521,6 @@ void briefing_editor_dlg::update_data(int update)
 			}
 			ptr->icons[m_last_icon].type = m_icon_image;
 
-			// if this icon has a special closeup image (like a jump node)
-			// then make sure the ship class doesn't have a special icon
-			if (brief_special_closeup(m_icon_image))
-				m_ship_type = get_first_ship_with_no_custom_icon();
```

Comment:
Do we still need this function, or can we get rid of it?

Reply:
We still need it.  It's still used in FSO, and it could be useful for other purposes in the future.

Reply:
Okay about potentially being useful in the future, but AFAICT `get_first_ship_with_no_custom_icon()` a protected function of this dialog class and is called just here.

Reply:
Ah, I thought you meant `brief_special_closeup`.